### PR TITLE
Edit account

### PR DESCRIPTION
**Pull request by D-Zaleski**
just added the required fields to edit account
delete function works
need to setup cloudinary for avatar

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,27 +0,0 @@
-require_relative "boot"
-
-require "rails/all"
-
```
Comment (syboulette) [inline config/application.rb:4]: Any reason why you deleted the configuration of the entire app?

**Review by syboulette — CHANGES_REQUESTED** [review]

**Review by syboulette — APPROVED** [review]

**Review by akramerin — COMMENTED** [review]
lgtm - except for the log/tmp -files. please remove!

**Comment by syboulette** [comment]
@D-Zaleski you still have a lot of bin files and the log devellopment

**Comment by syboulette** [comment]
@D-Zaleski Still a lot of cache

**Review by syboulette — APPROVED** [review]

**Pull request merged**